### ISSUE TITLE
How to sort on save?

### ISSUE THREAD

**Issue by yassun4dev** [pascalre/vscode-yaml-sort#38]
I'm trying to sort automatically on save like this... but cannot run.

```json:
// .vscode/settings.json

"[yaml]": {
      "editor.codeActionsOnSave": {
        "vscode-yaml-sort.sortYaml": true
      }
    },
```

Do you know a good way to sort files on save?

**Comment by pascalre** [comment]
@yassun4dev you need to register this extension as vs code formatter. Furthermore you must configure vs code to format files on save. Caution: This setting will apply for all files. To do so set
```
// .vscode/settings.json
{
  "editor.formatOnSave": true,
  "vscode-yaml-sort.useAsFormatter": true
}
```

**Comment by yassun4dev** [comment]
@pascalre I was able to format the file on save. Thanks!

But, I cannot sort. I need formatting options, including sorting.

**Comment by pascalre** [comment]
Oh sorry, I read that wrong! To sort on save I would need to add a further configuration, e. g. `sortOnSave`, and update a small part of the sorting function. I will definitely look into this in a while.

**Comment by yassun4dev** [comment]
Thank you! That would help a lot.

**Comment by pascalre** [comment]
Hello @yassun4dev, I just published version 5.3.0 which will add the setting `sortOnSave`. In the [README.md](https://github.com/pascalre/vscode-yaml-sort#how-to-sort-on-save) you can read how to sort on save.

**Comment by yassun4dev** [comment]
@pascalre

Thank you!!

I just applied it to my project and it worked very well.
